__tests__/app.test.js

```diff
@@ -181,21 +181,91 @@ describe("/api/articles/:article_id/comments", () => {
         });
       });
   });
-  test('400: returns an error when given a non-numeric article_id', () => {
+  test("400: returns an error when given a non-numeric article_id", () => {
+    return request(app)
+      .get("/api/articles/article4/comments")
+      .expect(400)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Bad Request: Article_id invalid");
+      });
+  });
+  test("404: returns an error when given a numeric article_id that doesn't exist", () => {
+    return request(app)
+      .get("/api/articles/193476/comments")
+      .expect(404)
+      .then(({ body }) => {
+        const { msg } = body;
+        expect(msg).toBe("Not Found: Article does not exist");
+      });
+  });
+});
+
+describe("POST /api/articles/:article_id/comments", () => {
+  test("201: adds a new comment to the table and responds with the comment", () => {
+    const newComment = {
+      body: "Comment McCommentface",
+      username: "butter_bridge",
+    };
+    
+    const desiredReturnedComment = {
+      body: "Comment McCommentface",
+      votes: 0,
+      author: "butter_bridge",
+      article_id: 1,
+      created_at: expect.any(String),
+      comment_id: 19
+    };
+
     return request(app)
-    .get('/api/articles/article4/comments')
+    .post("/api/articles/1/comments")
+    .send(newComment)
+    .expect(201)
+    .then(({body}) => {
+      const {comment} = body
+      expect(comment).toMatchObject(desiredReturnedComment)
+    })
+  });
```

Comment:
We should add a test for when there are unnecessary properties in the comment I.E. {fruit: 'apple', username: 'butter_bridge', body: 'this is a comment'}. We want to ignore the extra keys and send back 201 and the comment 🙂